app/views/network_router/_common_new_edit.haml

```diff
@@ -0,0 +1,71 @@
+%h3
+  = _('Basic Information')
+.form-horizontal
+  .form-group
+    %label.col-md-2.control-label
+      = _('Router Name')
+    .col-md-8
+      %input.form-control{:type          => "text",
+                          :name          => "name",
+                          'ng-model'     => "networkRouterModel.name",
+                          'ng-maxlength' => 128,
+                          :miqrequired   => true,
+                          :checkchange   => true}
+%h3
```

Comment:
Fields that are Required need to be explicitly shown as `Required` including the dropdowns below